libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateRangePicker.tsx

```diff
@@ -191,10 +214,14 @@ class DateRangePickerComponent extends React.Component<DateRangePickerProps, IDa
                     <div className="gd-date-range-picker datetime s-date-range-picker">
                         <label>{intl.formatMessage({ id: "filters.from" })}</label>
                         {FromField}
-                        {isFromInputDatePickerOpen ? DatePicker : null}
+                        {isFromInputDatePickerOpen ? (
+                            <>{shouldOverlayDatePicker ? OverlayDatePicker : DatePicker}</>
+                        ) : null}
                         <label>{intl.formatMessage({ id: "filters.to" })}</label>
                         {ToField}
-                        {isToInputDatePickerOpen ? DatePicker : null}
+                        {isToInputDatePickerOpen ? (
+                            <>{shouldOverlayDatePicker ? OverlayDatePicker : DatePicker}</>
```

Comment:
I would extract this as a constant to not repeat this block twice, i.e., 
`const DatePickerComponent = shouldOverlayDatePicker ? OverlayDatePicker : DatePicker;`

Possibly the empty fragment will not be necessary.

Reply:
Done